openstack_plugin/resources/compute/server.py

```diff
@@ -112,6 +112,7 @@ def _stop_server(server):
             server.stop()
             ctx.instance.runtime_properties[SERVER_TASK_STOP]\
                 = SERVER_ACTION_STATUS_PENDING
+            ctx.instance.update()
```

Comment:
@0lvin Why calling update instance context here is needed ?

Reply:
We need to save current state before run external call.